### PR TITLE
Give more visibility when creating release

### PR DESCRIPTION
**Pull request by tetienne**
- Implement release drafter
- Update github action version

### CODE REVIEW

**Comment by h4de5** [comment]
thank you again for your contribution. sorry that I cannot invest more time into reviewing it.

so far I think this is about how the release is shown in the github releases? something like building up the descriptions from the commits since the last release right?

there are three points I don't understand or that I am unsure if this will continue to work:
1. I know its better to have PRs and seperate branches for each feature - but would this change prevent me from creating a new release by editing code directly in the main branch through the github website?
2. As you removed the release.yaml - I can't see where the actual release is done. currently there is a condition tied on that manifest version. so a release is only created once that version changes. that way I can make multiple commits into main without creating a new release everytime.
3. And I would like to keep that minimal release zip file - that one that just includes the custom_component without any of the files in the root directory. I understand this is gone?

**Comment by h4de5** [comment]
in regards to your contributor request: I really welcome your changes and the quality you provide with them. I also agree that I do not have enough time on hand to review bugs or PRs, as I am also working on another HA integration here on github and some more oss projects. 
the problem I see is, that I do use those integrations by myself. this means any changes done here, are coming to my home sooner or later. and in the past, there has not been a single PR from any contributer that did not break something at home (even though it was tested and it should not break anything). that in turn creates even more work of course - work that I cannot postbone. thats also why I am really cautious with that stupid firmware update from toshiba. I have build up a test environment to test for new PRs but this alone is sometimes too time consuming. e.g. there is an open PR for the vimar integration I am "working" for months now.

Long story short: lets finish this firmware upate and maybe get the release process straight and I will add you as contributor here afterwards.
Thank you very much again for your work!

**Comment by chrisandsally** [comment]
I just want to say a massive thank you to everyone that has come together to fix the change Toshiba made, I wish I knew how to code, but a real sincere thanks for personal time that you folks put in for others.
